engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkDeploymentManager.scala

```diff
@@ -21,15 +26,28 @@ abstract class FlinkDeploymentManager(
     modelData: BaseModelData,
     dependencies: DeploymentManagerDependencies,
     shouldVerifyBeforeDeploy: Boolean,
-    mainClassName: String
+    mainClassName: String,
+    scenarioTestingConfig: ScenarioTestingConfig
 ) extends DeploymentManager
     with LazyLogging {
 
   import dependencies._
 
-  private lazy val testRunner = new FlinkProcessTestRunner(modelData.asInvokableModelData)
+  private val scenarioTestingMiniClusterWrapperOpt =
```

Comment:
Could we use some named trait/class instead of `AutoCloseable`, and bind our classes (like `FlinkProcessTestRunner`/`FlinkProcessVerifier` to the trait/class)

I would like to know what class calls the close() method now, but I can only see `AutoCloseable` usages.

Reply:
There is a problem with this. Flink executor and flink DM don't have shared module for such things. If they do have shared module, we could organize otherwise things related with reflection. But I don't think that it is time to invest into such refactor